android/app/build.gradle

```diff
@@ -184,6 +184,8 @@ android {
         resValue "string", "branch_io_app_link_alternate", "abgsz-alternate.test-app.link"
         resValue "string", "branch_io_key_live", ""
         resValue "string", "branch_io_key_test", "key_test_fdVQmXxlMInIG98ByzdqlbdkCxi8QO3a"
+
+        resValue "string", "instabug_token", ""
```

Comment:
Do we plan to support instabug in debug builds?

Reply:
On Android, it's the equivalent of setting Instabug token in `buildConfig.js`, because the initialisation differs between platforms.